tests/platform_tests/test_sfp.py

```diff
@@ -211,18 +211,28 @@ def test_check_sfp_low_power_mode(duthost, conn_graph_facts):
 
     logging.info("Try to change SFP lpmode")
     tested_physical_ports = set()
+    not_supporting_lpm_physical_ports = set()
     for intf in conn_graph_facts["device_conn"]:
         phy_intf = portmap[intf][0]
         if phy_intf in tested_physical_ports:
             logging.info("skip tested SFPs {} to avoid repeating operating physical interface {}".format(intf, phy_intf))
             continue
+        type = duthost.command('redis-cli -n 6 hget "TRANSCEIVER_INFO|{}" type'.format(intf))["stdout"]
```

Comment:
`type` will shadow the python built-in function. Suggest to use another variable name.

Reply:
Fixed.